### ISSUE TITLE
Don't delete entire project if setting "targets" to a string

### ISSUE THREAD

**Issue by zivpugatch** [DongShelton/rollup-plugin-clear#3]
When setting "targets" to a string instead of an array of strings, the entire project is deleted.

sample: 
```
{
    targets: './dist/'
}
```
might delete entire project from root.

**Comment by markcellus** [comment]
Yeah this happened to me quite a few times already. Glad it uses `process.cwd()` instead of the global root of my entire system! 😳

```bash
> rollup
cleared:  / 
```

😩
